tests/unittest_inference.py

```diff
@@ -706,14 +706,6 @@ class InvalidGetitem2(object):
         NoGetitem()[4] #@
         InvalidGetitem()[5] #@
         InvalidGetitem2()[10] #@
-        """
-        )
-        for node in ast_nodes[:3]:
-            self.assertRaises(InferenceError, next, node.infer())
-        for node in ast_nodes[3:]:
-            self.assertEqual(next(node.infer()), util.Uninferable)
```

Comment:
These lines do nothing since `ast_nodes[3:]` is empty

Reply:
Since we loop through all the nodes below:

```python
for node in ast_nodes:
    self.assertRaises(InferenceError, next, node.infer())
```

Reply:
:+1: